src/renderer/canvas/canvas-renderer.ts

```diff
@@ -362,6 +362,28 @@ export class CanvasRenderer implements ITableEngineRenderer {
 		this._initializeCellRenderers();
 	}
 
+	/**
+	 * Create a container Element in which the canvas and html view will later be injected.
+	 * @param parentContainer the container defined by the user
+	 */
+	private _createTableContainer(parentContainer: HTMLElement): HTMLElement {
+		const div = document.createElement('div');
+		// full size in parent
+		div.style.width = '100%';
+		div.style.height = '100%';
+		// Allow absolute positioning within the container
+		div.style.position = 'relative';
+		// replace possilbe global styles.
+		div.style.padding = '0';
+		div.style.border = '0';
```

Comment:
I think we should remove the last two lines since it's kind of unexpected (at least for me). It may be better to leave that to the developers using the library. What do you think?

Reply:
Since this is an new element,  which is not in control of the user,  I think this okay.
The goal is that  things like following CSS will not break the table. 
` div { padding: 10px; } `

Reply:
I see, let's keep it for now 👍